static/app/views/setupWizard/index.tsx

```diff
@@ -192,6 +208,17 @@ function ProjectSelection({hash, organizations = []}: Omit<Props, 'allowSelectio
     [orgProjectsRequest.data]
   );
 
+  const {options: cachedProjectOptions} = useCompactSelectOptionsCache(projectOptions);
+
+  const handleSearch = useMemo(
+    () =>
+      debounce((value: string) => {
```

Comment:
Can we use already existing hook for debounce value from `static/app/utils/useDebouncedValue.tsx`

Reply:
good point! forgot that it exists